scripts/system/more/more.html

```diff
@@ -0,0 +1,225 @@
+<!--
+//  more.html
```

Comment:
Don't need `//`s.

Reply:
This is how it is done everywhere else in the code base, likely for easy copying and pasting through to file types that don't support wrap-around header commenting.